lib/Command/ExpirePassword.php

```diff
@@ -83,17 +95,28 @@ protected function configure() {
 			->setDescription('Expire a user\'s password')
 			->addArgument(
 					'uid',
-					InputArgument::REQUIRED,
+					InputArgument::OPTIONAL,
 					'The user\'s ownCloud uid (username).'
-				     )
+				)
 			->addArgument(
 					'expiredate',
 					InputArgument::OPTIONAL,
 					'The date and time when a password expires, e.g. "2019-01-01 14:00:00 CET".',
 					'-1 days'		// base.php sets timezone to utc, so
 										// make sure date is in the past
-				     )
-		;
+				)
+			->addOption(
+				'all',
+				null,
+				InputOption::VALUE_NONE,
+				'Will add password expiry to all known users.'
+			)
+			->addOption(
+				'groups',
```

Comment:
In ``core``  repo ``core/Command/User/Add.php`` the option is called just ``group`` because you specify just 1 group name after each ``--group`` : 
```
--group Finance,Sydney --group HR,Sydney --group Finance,Berlin
```
Should the option here be just ``group`` ?

Reply:
Renamed it to `group`.